### PR TITLE
sl-259: The new project should displayed first

### PR DESCRIPTION
**Pull request by Izdormu**

### CODE REVIEW

**Review by andanf-e — CHANGES_REQUESTED** [review]
Just add ordering by createdAt at the backend side

File: backend/src/packages/projects/project.repository.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ class ProjectRepository implements Omit<IRepository, 'update' | 'delete'> {
           void builder.where('name', 'ilike', `%${name}%`);
         }
       })
-      .orderBy('created_at')
+      .orderBy('created_at', 'desc')
```
Comment (andanf-e) [inline backend/src/packages/projects/project.repository.ts]: Create a enum for sorting order

File: backend/src/libs/packages/database/libs/enums/database-sort-direction.enum.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const DatabaseSortDirection = {
```
Comment (what1s1ove) [inline backend/src/libs/packages/database/libs/enums/database-sort-direction.enum.ts]: move it to the shared folder

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: shared/src/packages/projects/libs/enums/projects-sort-direction.enum.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const ProjectsSortDirection = {
+  DESCENDING: 'desc',
+} as const;
```
Comment (what1s1ove) [inline shared/src/packages/projects/libs/enums/projects-sort-direction.enum.ts]:
```suggestion
const SortDirection = {
  DESCENDING: 'desc',
} as const;
```
it is a common enum

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: backend/src/packages/projects/libs/enums/enums.ts [hunk 4]
```diff
@@ -1 +1,5 @@
-export { ProjectCategory, ProjectsApiPath } from 'shared/build/index.js';
+export {
+  ProjectCategory,
+  ProjectsApiPath,
+  SortDirection,
```
Comment (what1s1ove) [inline backend/src/packages/projects/libs/enums/enums.ts]:
```suggestion
```
it should be imported in the common enums

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**